sales_team_security/models/res_partner.py

```diff
@@ -22,8 +82,8 @@ def _remove_key_followers(self, partner):
 
     def _add_followers_from_salesmen(self):
         """Sync followers in commercial partner + delivery/invoice contacts."""
-        for record in self.commercial_partner_id:
-            followers = (record.child_ids + record).user_id.partner_id
+        for record in self.mapped("commercial_partner_id"):
```

Comment:
Those changes are not necessary as Odoo already include a `mapped()` when you access a relation through a multi recordset